README.md

```diff
@@ -9,11 +9,22 @@
 
 # *pydicom*
 
-*pydicom* is a pure Python package for working with [DICOM](https://www.dicomstandard.org/) files. It lets you read, modify and write DICOM data in an easy "pythonic" way.
+*pydicom* is a pure Python package for working with [DICOM](https://www.dicomstandard.org/) files.
+It lets you read, modify and write DICOM data in an easy "pythonic" way. As a pure Python package,
+*pydicom* can run anywhere Python runs without any other requirements, although if you're working 
+with *Pixel Data* then we recommend you also install [NumPy](http://www.numpy.org).
 
-As a pure Python package, *pydicom* can run anywhere Python runs without any other requirements, although if you're working with *Pixel Data* then we recommend you also install [NumPy](http://www.numpy.org).
+Note that *pydicom* is a general-purpose DICOM framework concerned with 
+reading and writing DICOM datasets, and does not handle the specifics
+of individual SOP classes or other aspects of DICOM on purpose. Other 
+libraries both inside and outside the [pydicom organization](https://github.com/pydicom)
```

Comment:
The "on purpose" here sounds a bit odd to me.  Perhaps needs more explanation, like '...or other aspects of DICOM, in order to keep the project manageable'.